release/src/main/groovy/run_python_mobile_gaming.sh

```diff
@@ -0,0 +1,166 @@
+#!/bin/bash
+#
+#    Licensed to the Apache Software Foundation (ASF) under one or more
+#    contributor license agreements.  See the NOTICE file distributed with
+#    this work for additional information regarding copyright ownership.
+#    The ASF licenses this file to You under the Apache License, Version 2.0
+#    (the "License"); you may not use this file except in compliance with
+#    the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+#  This file will verify Apache/Beam release candidate python by following steps:
+#
+#  1. Create a new virtualenv and install the SDK
+#  2. Run UserScore examples with DirectRunner
+#  3. Run UserScore examples with DataflowRunner
+#  4. Run HourlyTeamScore on DirectRunner
+#  5. Run HourlyTeamScore on DataflowRunner
+#
+
+set -e
+set -v
+
+source release/src/main/groovy/python_release_automation_utils.sh
+
+complete() {
+    print_separator "Validation $1"
+    rm -rf $TMPDIR
+}
+
+verify_houly_team_score() {
+    # $1 runner type
+    retry=3
+    should_see='AntiqueBrassPlatypus'
+    while(( $retry >= 0 ))
+    do
+        if [[ $retry > 0 ]]
+        then
+            bq_pull_result=$(bq head -n 100 $DATASET.hourly_team_score_python_$1)
+            if [[ $bq_pull_result = *"$should_see"* ]]
+            then
+                echo "SUCCEED: HourlyTeamScore example successful run on $1-runner"
+                break
+            else
+                retry=$(($retry-1))
+                echo "Did not find team scores, retry left: $retry"
+                sleep 15
+            fi
+        else
+            echo "FAILED: HourlyTeamScore example failed running on $1-runner. Did not found scores of team $should_see in $DATASET.leader_board"
+            complete "FAILED"
+            exit 1
+        fi
+    done
+}
+
+
+print_separator "Start Mobile Gaming Examples"
+echo "SDK version: $VERSION"
+
+TMPDIR=$(mktemp -d)
+echo $TMPDIR
+pushd $TMPDIR
+
+#
+# 1. Download files from RC staging location, install python sdk
+#
+
+wget $CANDIDATE_URL$BEAM_PYTHON_SDK
```

Comment:
Since this file is not release specific, should it build and run SDKs at head?

Reply:
run_python_mobile_gaming.sh is release specific at this moment (I should find a better name for this file that stop confusing people). In the future, we want to run this test against Python SDK nightly snapshot once we have snapshot ready.